package.json

```diff
@@ -17,14 +22,23 @@
   "author": "Jeff Escalante",
   "license": "MPL-2.0",
   "dependencies": {
-    "body-parser": "^1.19.0",
     "chalk": "^4.0.0",
     "chokidar": "^3.4.0",
     "commander": "^5.0.0",
     "express": "^4.17.1"
   },
+  "peerDependencies": {
+    "next": ">=10"
```

Comment:
@jescalan would this count as _range_ ? As per our discussion, this should include all versions major than next 10?

Reply:
yeah this is perfect!